bedms/utils.py

```diff
@@ -167,15 +234,58 @@ def data_encoding(
         with open(lb_path, "rb") as f:
             label_encoder = pickle.load(f)
 
+    elif schema == "BEDBASE":
+        vectorizer = CountVectorizer()
+        vc_path = hf_hub_download(repo_id=REPO_ID, filename=BEDBASE_VECTORIZER_FILENAME)
+        with open(vc_path, "rb") as f:
+            vectorizer = pickle.load(f)
```

Comment:
Why instantiate a blank one just to override it?

Reply:
Also, am I correct in that you are calling the unpickling for each PEP processed? I think it'd be more efficient to unpickle in `AttrStandarizer._load_model` once and use those instead of calling `hf_hub_download` and `pickle.load` many times